obspy/geodetics/tests/test_util_geodetics.py

```diff
@@ -110,6 +109,72 @@ def test_calcVincentyInverse2(self):
         self.assertAlmostEqual(alpha12, calc_alpha12)
         self.assertAlmostEqual(alpha21, calc_alpha21)
 
+    def test_calcVincentyInverseTabulated(self):
+        """ Tabulated results for Vincenty Inverse
+
+        Table II of Vincenty's paper (T. Vincenty 1975, "Direct
+        and inverse solutions of geodesics on the ellipsoid with
+        application of nested equations" Survey Review XXII
+        pp.88-93) has five test examples for the forward and inverse
+        problem (with results rounded to 0.00001 seconds of arc and
+        1 mm). The inverse versions of these are implemented here. Note
+        the non-standard (old) ellipsoid usage. Here we test that we
+        match these examples for the inverse problem.
+        """
+        # Row "A"
+        # NB: for this case there seems to be a typo in
+        #     the tabulated data. Tabulated data is commented
+        #     out and values from geographiclib are used in there place
+        # dist = 14110.526170
+        dist = 14039.003954192352
```

Comment:
Is there a reason to use kilometres instead of metres like the docstring/paper/etc.?